rust/connlib/libs/tunnel/src/lib.rs

```diff
@@ -224,17 +227,7 @@ where
                 }
             }
         }
-        let resource_list = {
-            let mut resources = self.resources.write();
-            resources.insert(resource_description);
-            resources.resource_list()
-        };
-        match resource_list {
-            Ok(resource_list) => {
-                self.callbacks.on_update_resources(resource_list);
-            }
-            Err(err) => self.callbacks.on_error(&err.into(), Fatal),
-        }
```

Comment:
Why was this removed?

Reply:
My bad, this happened when rebasing and I didn't see it.